src/System.IO.IsolatedStorage/tests/System.IO.IsolatedStorage.Tests.builds

```diff
@@ -3,15 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.IO.IsolatedStorage.Tests.csproj">
-      <OSGroup>Windows_NT</OSGroup>
```

Comment:
Ok, I'll roll it back for now. It seems less than optimal to have three separate builds that all build the same binary. In addition we're really building AnyCPU, which makes this pretty confusing. It would make a lot more sense if the TestTFM figured into the actual test directory IMHO.